### PR TITLE
Issue #170: Org Details Map and Program Cards

### LINKED ISSUE #170: Listing of Program cards  and Org Map (organization screen)

### PR DESCRIPTION
**Pull request by mckoon**
* Adds MapQuery to organization-details.
* Shows map on organization-details.
* Shows program-cards on organization-details.
* Creates program-card component.

Closes #170

### CODE REVIEW

**Comment by mckoon** [comment]
See this in action at https://hmhb-mckoon.herokuapp.com/views/organization-details/2

**Comment by mckoon** [comment]
# Screenshots

## Program Screen

screen shot 2016-11-02 at 8 41 39 pm

## Organization Screen, now with Programs

screen shot 2016-11-02 at 8 40 19 pm

screen shot 2016-11-02 at 8 40 36 pm

screen shot 2016-11-02 at 8 40 57 pm

screen shot 2016-11-02 at 8 41 06 pm

File: client/src/main/webjar/organization-form-dialog/organization-form-dialog.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
            flex="80">
 
     <form name="organizationFormDialog.organizationForm"
-             ng-submit="organizationFormDialog.save()">
```
Comment (RyanBard) [inline client/src/main/webjar/organization-form-dialog/organization-form-dialog.html]: about time you fixed that :)

Comment (mckoon) [inline client/src/main/webjar/organization-form-dialog/organization-form-dialog.html]: Yeah, was like that for almost 10 hours. Really dodged a bullet.

**Pull request merged**